code/parse/sexp.cpp

```diff
@@ -26817,6 +26830,10 @@ int query_operator_argument_type(int op, int argnum)
 			else
 				return OPF_SHIP_WING;
 
+		case OP_SECONDS_TO_GOAL:
```

Comment:
Flagged by CI.  This cases a fall-through case when argnum != 0.  Briefly looking at the other cases doesn't give me a clear idea of whats supposed to happen here, obviously something needs to be returned as the else statement, but I don't know what that should be at this time.